src/client/pages/OfferNew/Checkout/index.tsx

```diff
@@ -338,9 +343,15 @@ export const Checkout = ({
   const onPhoneChange = async (phoneNumber: string) => {
     const { phoneNumber: currentPhone } = offerData.person
     if (!phoneNumber || currentPhone === phoneNumber) return
+    setIsPhoneNumberUpdating(true)
     await editQuotes(quoteIds, { phoneNumber })
+    setIsPhoneNumberUpdating(false)
```

Comment:
What happens if there's a network error here? Should we have a catch/finally block that always makes sure to `setIsPhoneNumberUpdating(false)`?

Reply:
I guess if there is a network error you would not be able to add the ssn so you cannot sign at all. 😞 
Right ?

Reply:
Yeah it's an edge case perhaps but it could be a temporary issue and then we want users to be able to retry and send the request again, right?